dubbo-remoting/dubbo-remoting-netty/src/main/resources/META-INF/dubbo/internal/com.alibaba.dubbo.remoting.Transporter

```diff
@@ -1 +1 @@
-netty=com.alibaba.dubbo.remoting.transport.netty.NettyTransporter
\ No newline at end of file
+netty3=com.alibaba.dubbo.remoting.transport.netty.NettyTransporter
```

Comment:
This change may impact some user, especially for user who is writing code base on Dubbo SPI. This change need highlight this change in RELEASE or docs.

Reply:
the default implementation named `netty` is in dubbo-remoting-netty4 module

Reply:
I think we still need to mention this change in the release notes. Netty4 is now the default.